blog/2024-02-29-boa-release-18/index.md

```diff
@@ -18,361 +18,114 @@ able to use.
 
-// `main.mjs` or any of its imports could import `main.mjs` itself, so we
-// insert it into the loader for good measure.
-loader.insert(Path::new("main.mjs").to_path_buf(), module.clone());
+This is a great achievement for us and we are very proud to be listed alongside other major JavaScript engines. It should be much easier for users to compare Boa's conformance tests with other engines.
 
-// All modules use promises to signal completion of its lifecycle.
-// The utility method `load_link_evaluate` calls `load`, then `link` and
-// finally `evaluate`, returning an error if any call fails.
-let promise = module.load_link_evaluate(context).unwrap();
+Below is a screenshot showing Boa being 83% conformance with the latest tests and to the right statistics on the time taken to performa a full test run in comparison to other engines.
 
-// Important to push the job queue forward! Otherwise, the modules won't progress
-// on their lifecycle.
-context.run_jobs();
+![img](./conformance_dark.webp)
 
-// All modules return `undefined` if they're successfully evaluated.
-assert_eq!(promise.state().unwrap(), PromiseState::Fulfilled(JsValue::undefined()));
-```
-
-For a more extensive, descriptive example that uses a real directory, you can check out [**boa_examples**][mod_example].
-
-[modules]: https://tc39.es/ecma262/#sec-modules
-[mod_loader]: https://docs.rs/boa_engine/0.17.0/boa_engine/module/trait.ModuleLoader.html
-[mod_example]: https://github.com/boa-dev/boa/blob/v0.17/boa_examples/src/bin/modules.rs
-
-### Spec Version Conformance
-
-Something we get asked a lot is _"Do you support ES5 or ES6"_? or _"How far away are you from supporting ESX"_?
-We're pleased to say we've updated our conformance board to show you how we're doing across ES versions.
+### Temporal
 
-Just navigate to our [Test262 Dashboard][conformance], select "Test Results" on our main branch, and
-then you can use the dropdown underneath to see how we're doing on each version. ES5 and ES6 are very close, you can see
-we're only a few tests away from them being fully implemented.
+The majority of our work over the past few months has been on the Temporal API. The Temporal API is a new set of built-in objects and functions that provide a modern, ECMAScript language standard for date and time manipulation. The Temporal API is designed to be a more modern replacement for the `Date` object, and it provides a more feature-rich and flexible API for working with dates and times.
 
-### Optimizations
+It is currently a [stage 3 proposal](https://tc39.es/proposal-temporal/docs/) and we are working alongside the TC39 champions to put together a solid implementation. As Temporal is such a large implementation, we have done most of the work outside of Boa so that it can be used in other projects. This work can be found in the [temporal_rs](https://github.com/boa-dev/temporal/) repository.
```

Comment:
```suggestion
It is currently a [stage 3 proposal](https://tc39.es/proposal-temporal/docs/) and we are working alongside the TC39 champions to put together a solid implementation. Since Temporal is such an extensive specification, we have done most of the work outside of Boa so that it can be used in other projects. This work can be found in the [temporal_rs](https://github.com/boa-dev/temporal/) repository.
```